server-adapters/resteasy-reactor-netty/src/main/java/org/jboss/resteasy/plugins/server/reactor/netty/i18n/Messages.java

```diff
@@ -0,0 +1,39 @@
+package org.jboss.resteasy.plugins.server.reactor.netty.i18n;
+
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageBundle;
+
+/**
+ *
+ * @author <a href="ron.sigal@jboss.com">Ron Sigal</a>
+ * @version $Revision: 1.1 $
+ *
+ * Copyright Sep 1, 2015
+ */
+@MessageBundle(projectCode = "RESTEASY")
+public interface Messages
+{
+    Messages MESSAGES = org.jboss.logging.Messages.getBundle(Messages.class);
+    int BASE = 22500;
+
+    @Message(id = BASE + 0, value = "Already committed")
+    String alreadyCommitted();
```

Comment:
I realize this is just copied from another instance, but I think we should just make the return type `IllegalStateException` and in the `ReactorNettyHttpResponse` we just do `throw Messages.MESSAGES.alreadyCommitted()`.

Reply:
Hmm, is there precedent for that?  I can make this change, but it seems a little to me.  If I hear nothing else, I will make the change.

Reply:
Looking at most of RESTEasy it seems strings are used. However I'm personally in favor of just constructing the exception in the logging tooling. I have not strong opinion. At some point I might look at changing this in other places so if it stays like this that is fine too :)